Website/Sources/ConferenceWebsite.swift

```diff
@@ -61,6 +62,36 @@ struct ConferenceWebsite {
       }
     }
   }
+
+  private static func copyAASAFile() throws {
+    let fileManager = FileManager.default
+
+    let websiteDirectory = try URL.selectDirectories(from: #file).source
+    let websiteWellKnownDirectory = websiteDirectory.appending(path: ".well-known")
+    let aasaFile = websiteWellKnownDirectory.appending(path: "apple-app-site-association")
+
+    if !fileManager.fileExists(atPath: websiteWellKnownDirectory.path) {
+      try fileManager.createDirectory(
+        at: websiteWellKnownDirectory,
+        withIntermediateDirectories: true,
+        attributes: nil
+      )
+    }
+
+    if fileManager.fileExists(atPath: aasaFile.path) {
+      try fileManager.removeItem(at: aasaFile)
+    }
+
+    let aasaContent = """
+    {
+        "appclips": {
+            "apps": ["jp.tryswift.Clip"]
```

Comment:
I think I might need to make the following changes. 💭 
Since I haven’t released an App Clip to production before, there’s a chance I’m misunderstanding something. 🙇‍♂️ 
```suggestion
            "apps": ["9PC9DZ9559.jp.tryswift.tokyo.App.Clip"]
```
ref:
[Supporting associated domains](https://developer.apple.com/documentation/xcode/supporting-associated-domains)

https://github.com/tryswift/try-swift-tokyo/blob/364db2a50051310a01bfe020ed17061dc06a1698/App/App.xcodeproj/project.pbxproj#L467

https://github.com/tryswift/try-swift-tokyo/blob/364db2a50051310a01bfe020ed17061dc06a1698/App/App.xcodeproj/project.pbxproj#L556